Car_Localizer/Car_Localizer/Base_loss_model/loss_model_base.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class LossBase(nn.Module):
+    def __init__(self):
+        super(LossBase, self).__init__()
```

Comment:
since python 3.8 you do need to pass explicitly a class name to `super()` method
super().__init__()